A bit more nuance in the motivation (#8)
DNS-Hackathon/catalog-extensions-draft: draft-axu-dnsop-catalog-zone-xfr-properties.md

{{!RFC9432 (DNS Catalog Zones)}} described a method for automatic DNS zone provisioning among DNS name servers by the catalog of zones to be provisioned as one or more regular DNS zones.
Configuration associated with the member zones, such as from which primary name servers and with which {{!RFC8945 (TSIG keys)}} to transfer the zones, and from which IP addresses and with which TSIG keys {{!RFC1996 (DNS notifies)}} are allowed, were assumed to be preprovisioned at the catalog consumer.

> This document specifies DNS Catalog Zones Properties to specify primary name servers and TSIG keys to use to transfer the member zones in a catalog, as well as properties to specify which IP addresses, using which TSIG keys, are allowed to notify {{!RFC1996}} the secondary name server serving the member zones, in order to:

Reviewer: This document specifies DNS Catalog Zones Properties to specify primary name servers to transfer the member zones in a catalog from, as well as properties to specify which IP addresses, using which cryptographic keys, are allowed to notify {{!RFC1996}} the secondary name server serving the member zones, in order to:

Author: Done in dc6b6de

> This document specifies DNS Catalog Zones Properties to specify primary name servers and TSIG keys to use to transfer the member zones in a catalog, as well as properties to specify which IP addresses, using which TSIG keys, are allowed to notify {{!RFC1996}} the secondary name server serving the member zones, in order to:

Author: ```suggestion
This document specifies DNS Catalog Zones Properties to specify primary name servers to transfer the member zones in a catalog from, as well as properties to specify which IP addresses, using which cryptographic keys, are allowed to notify {{!RFC1996}} the secondary name server serving the member zones, in order to:
```
